bundles/framework/personaldata/PublishedMapsList.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import { Message, Confirm } from 'oskari-ui'
+import { Table } from 'oskari-ui/components/Table'
+import { DeleteOutlined, EditOutlined } from '@ant-design/icons'
+import { red } from '@ant-design/colors'
+import styled from 'styled-components';
+import { showHTMLrender } from './view/HTMLrender/HTMLrender'
+
+const DELETE_ICON_STYLE = {
+    color: red.primary
+};
+
+const StyledTable = styled(Table)`
+    a {
+        cursor: pointer;
+    }
+`
+
+const BUNDLE_NAME = 'PersonalData';
+
+export const PublishedMapsList = ({ views, handleEdit, handleDelete, handlePublish, showOnMap, setPopup, closePopup }) => {
+
+    const showHtml = (view) => {
+        const url = Oskari.getSandbox().createURL(view.url);
+        const size = view.metadata && view.metadata.size ? view.metadata.size : undefined;
+        const width = size ? size.width + 'px' : '100%';
+        const height = size ? size.height + 'px' : '100%';
+        let iframeCode = '<iframe src="' + url + '" allow="geolocation" style="border: none;';
+        if (width !== null && width !== undefined) {
+            iframeCode += ' width: ' + width + ';';
+        }
+        if (height !== null && height !== undefined) {
+            iframeCode += ' height: ' + height + ';';
+        }
+        iframeCode += '"></iframe>';
+        const values = {
+            html: iframeCode
+        };
+        
+        const controls = showHTMLrender(values, closePopup);
+        setPopup(controls);
+    }
+
+    const openView = (view) => {
+        window.open(
+            view.url,
+            'Published',
+            'location=1,status=1,scrollbars=yes,width=850,height=800'
+        );
+    }
+
+    const columnSettings = [
+        {
+            align: 'left',
+            title: <Message messageKey='tabs.publishedmaps.grid.name' bundleKey="PersonalData" />,
+            dataIndex: 'name',
+            render: (title, item) => {
+                return (
+                    <a onClick={() => openView(item)}>{title}</a>
+                );
+            }
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tabs.publishedmaps.grid.domain' bundleKey="PersonalData" />,
+            dataIndex: 'pubDomain',
+            render: (title, item) => {
+                return (
+                    <span>{title}</span>
+                )
+            }
```

Comment:
Might not need to create a render function for this? Try removing it and see if it works without it.